lib/rmt/cli/repos.rb

```diff
@@ -1,13 +1,17 @@
 class RMT::CLI::Repos < RMT::CLI::Base
 
-  default_task :list
```

Comment:
@tmuntan1 Any reason why you removed this? I'm not insisting on keeping it. One could argue that it makes it harder to get to the help output... but anyway, we should keep it consistent with how `products` command behaves.

Reply:
the default task hides that there are subcommands under it.